pkg/cmd/cli/cmd/rollback.go

```diff
@@ -210,24 +213,31 @@ func (o *RollbackOptions) Run() error {
 			return err
 		}
 		target = deployment
+		configName = r.Name
```

Comment:
if the object is not DC nor RC, do we continue? if not the default should error?